netmiko/fortinet/fortinet_ssh.py

```diff
@@ -166,9 +166,9 @@ def _get_output_mode_v6(self) -> str:
         pattern = r"^\s+set output (?P<mode>\S+)\s*$"
         result_mode_re = re.search(pattern, output, flags=re.M)
         if result_mode_re:
-            result_mode = result_mode_re.group("mode").strip()
-            if result_mode in ["more", "standard"]:
-                return result_mode
+            return "more"
+        else:
+            return "more"
```

Comment:
I am confused as to why you made this change? Why did you make any changes to the non-IBM driver?